mod-source-record-storage-server/src/main/java/org/folio/services/handlers/actions/AbstractUpdateModifyEventHandler.java

```diff
@@ -88,6 +88,7 @@ public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload p
             remove003FieldIfNeeded(changedRecord, hrId);
           }
           increaseGeneration(changedRecord);
+          changedRecord.setOrder(0);
```

Comment:
Hello @AivarIUsupov, I assume there might be a scenario during importing a file with multiple records using profile for record update, the all updated records from the file will have order 0 and the records in the log as well. Propose to set the order (a record number) of incoming record into updated record to keep the order of records from incoming file.